PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/AzureFunctionsUtils.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+package com.microsoft.intellij.util;
+
+import com.google.common.io.Files;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.SystemUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AzureFunctionsUtils {
+    public static File createMavenProjectToTempFolder(final String groupId, final String artifactId,
+            final String version, final String packageName, final String appName)
+            throws IOException, InterruptedException {
+        final boolean isWindows = isWindows();
+        final List<File> mvnCmds = resolvePathForCommand(isWindows ? "mvn.cmd" : "mvn", true);
+        if (!mvnCmds.isEmpty()) {
+            final File mvnCmd = mvnCmds.get(0);
+            final File folder = Files.createTempDir();
+            final Map<String, String> maps = new LinkedHashMap<>();
+            maps.put("-DarchetypeGroupId", "com.microsoft.azure");
+            maps.put("-DarchetypeArtifactId", "azure-functions-archetype");
+            maps.put("-DgroupId", groupId);
+            maps.put("-DartifactId", artifactId);
+            maps.put("-Dversion", version);
+            maps.put("-Dpackage", packageName);
+            maps.put("-DappName", appName);
```

Comment:
`appName` has been removed from function archetype, the default value is `${artifactId}-${timestamp}`
`